rubocop-sequel.gemspec

```diff
@@ -13,13 +13,13 @@ Gem::Specification.new do |gem|
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.name          = 'rubocop-sequel'
   gem.require_paths = ['lib']
-  gem.version       = '0.0.6'
+  gem.version       = '0.1.0'
 
-  gem.add_runtime_dependency 'rubocop', '~> 0.55', '>= 0.55'
+  gem.add_runtime_dependency 'rubocop', '~> 1.0', '>= 1.0.0'
```

Comment:
'~> 1.0'
Ties it to `rubocop` 1.0.x, while 1.1 is already out.
There should be no breaking changes in 1.x in the extensions API (check https://github.com/rubocop-hq/rubocop/pull/8991), so I guess it should be safe for you to use `>= 1.0`.

Reply:
From https://bundler.io/gemfile.html:
> ~> 2.1 is identical to >= 2.1 and < 3.0

So `~> 1.0` is fine for the `1.x` rubocop releases. I've tested this locally with `bundle install`.

(I don't know why all the dependencies use both ~> and >= - I'm just sticking with the existing style)

Reply:
Oh, you're right. I keep forgetting this syntax.

> (I don't know why all the dependencies use both ~> and >= - I'm just sticking with the existing style)

:shrug: :smile:

Reply:
So for this it makes sense `sqlite3', '~> 1.3', '>= 1.3.12'` since `~> 1.3` would match `1.3.0`, which isn't desired. However, I don't see any point in doing this `'sequel', '~> 4.49', '>= 4.49.0'` and there isn't a point in doing it for rubocop or rubocop-rspec either.